Libplanet.Net/Swarm.MessageHandlers.cs

```diff
@@ -186,26 +190,39 @@ private void ProcessBlockHeader(Message message)
 
         private async Task TransferTxsAsync(Message message)
         {
-            var getTxsMsg = (GetTxsMsg)message.Content;
-            foreach (TxId txid in getTxsMsg.TxIds)
+            if (!(_transferTxsSemaphore is null) &&
+                !_transferTxsSemaphore.WaitOne(TimeSpan.Zero))
+            {
+                return;
```

Comment:
Noticing the remote peer about this resource outage might be helpful. (of course, it causes another response type).